scripts/init.lisp

```diff
@@ -12,7 +12,8 @@
          '(null
            (english 1) ;; (language-name preferred-font-size)
            (chinese 2)
-           (russian 1))
+           (russian 1)
+           (italian 1))
```

Comment:
The list of languages is defined in a lisp script, which the game evaluates at startup. The engine concatenates the symbol from the list with a file extension, and loads the <language>.txt file from the rom.